openmp/runtime/src/z_Linux_util.cpp

```diff
@@ -2736,8 +2736,7 @@ int __kmp_get_load_balance(int max) {
 
 #endif // USE_LOAD_BALANCE
 
-#if !(KMP_ARCH_X86 || KMP_ARCH_X86_64 || KMP_MIC ||                            \
-      ((KMP_OS_LINUX || KMP_OS_DARWIN) && KMP_ARCH_AARCH64) ||                 \
+#if !(KMP_ARCH_X86 || KMP_ARCH_X86_64 || KMP_MIC || KMP_ARCH_AARCH64 ||        \
```

Comment:
I was a bit worried about this change, as it didn't have any `KMP_OS_WINDOWS` in it - but this file (labelled Linux) isn't compiled in Windows builds - contrary to `z_Linux_asm.S` which is used for Windows as well these days...

Reply:
The last part is confusing and I had submitted another PR wanting to rename this file to something more agnostic.